bacon/model/SpendingStatus.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  SpendingStatus.swift
+//  bacon
+//
+//  Created by Lizhi Zhang on 13/4/19.
+//  Copyright © 2019 nus.CS3217. All rights reserved.
+//
+
+import Foundation
+
+struct SpendingStatus {
+    let currentSpending: Decimal
+    let totalBudget: Decimal
+    var percentage: Decimal {
+        return currentSpending / totalBudget
```

Comment:
What if totalBudget == 0?

Reply:
Now it's changed to:
if total and current are both zero, return percentage 1 (so that we get a neutral pig)
if total is zero and current is not, return percentage 100 (just a big number, so we get a very sad pig)